LuaUI/Widgets/cmd_orbit.lua

```diff
@@ -21,12 +22,48 @@ local spTraceScreenRay = Spring.TraceScreenRay
 --------------------------------------------------------------------------------
 --------------------------------------------------------------------------------
 
+options_path = 'Settings/Interface/Circle Guard'
+
+options_order = {
+	'GuardOverrideOption',
+	'GuardRankOption'
+}
+
+options = {
+	GuardOverrideOption = {
+		name = "Global overrides selection rank",
+		desc = "Units issued a guard command will be treated as a different selection rank.",
+		type = "bool",
+		value = true,
+		noHotkey = true,
+		OnChange = function (self)
+	        WG.Orbit.IsSelectionOverrideSet = self.value
```

Comment:
some bad whitespace here
```suggestion
			WG.Orbit.IsSelectionOverrideSet = self.value
```